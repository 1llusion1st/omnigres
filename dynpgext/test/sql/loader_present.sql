-- There should be no loader
SHOW dynpgext.loader_present;
-- And dynpgext should indicate the same
SELECT loader_present();