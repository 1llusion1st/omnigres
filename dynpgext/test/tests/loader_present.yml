instances:
  default:
    init:
    - create extension dynpgext_test

tests:
- name: There should be no loader
  query: select count(*) from pg_settings where name = 'dynpgext.loader_present';
  results:
  - count: 0

- name: And dynpgext should indicate the same
  query: select loader_present()
  results:
  - loader_present: false