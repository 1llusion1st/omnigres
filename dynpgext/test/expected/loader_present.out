-- There should be no loader
SHOW dynpgext.loader_present;
ERROR:  unrecognized configuration parameter "dynpgext.loader_present"
-- And dynpgext should indicate the same
SELECT loader_present();
 loader_present 
----------------
 f
(1 row)

